src/Integration.Vsix/Commands/HelpMenu/AboutDialog.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<platformUi:DialogWindow x:Class="SonarLint.VisualStudio.Integration.Vsix.Commands.AboutDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    Title="About SonarLint"
+    ResizeMode="NoResize"
+    SizeToContent="WidthAndHeight"
+    Background="WhiteSmoke">
+    <Window.Resources>
+        <Style x:Key="TextBlocks" TargetType="TextBlock">
+            <Setter Property="FontSize" Value="12"/>
+            <Setter Property="Foreground" Value="Black"/>
+        </Style>
+    </Window.Resources>
+    <Grid Margin="8,8,8,8">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+        <Image Source="..\..\Resources\sonarlint_nobg.png" Grid.Row="0" Grid.Column="0" Stretch="None"/>
+        <Grid Grid.Row="0" Grid.Column="1" Margin="15,10,0,0">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+            </Grid.RowDefinitions>
+            <TextBlock Grid.Row="0" Text="Clean As You Code" Style="{StaticResource TextBlocks}" FontWeight="Bold"/>
+            <TextBlock Grid.Row="1" Text="{Binding SLVersion}" Style="{StaticResource TextBlocks}"/>
+            <TextBlock Grid.Row="2">
+                <Hyperlink NavigateUri="https://www.sonarsource.com/products/sonarlint/" RequestNavigate="ViewWebsite">
+                   <TextBlock Text="Visit our Website" Style="{StaticResource TextBlocks}"/>
+                 </Hyperlink>
+            </TextBlock>
+        </Grid>
+        <TextBlock Text="Copyright © 2016-2022 SonarSource SA" Grid.Row="1" Grid.Column="0" VerticalAlignment="Center" HorizontalAlignment="Left" Foreground="Gray" Style="{StaticResource TextBlocks}" Margin="0,8,0,0"/>
+        <platformUi:DialogButton Content="Close" Grid.Row="1" Grid.Column="1" IsCancel="True" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="0,8,0,0"/>
```

Comment:
I looked at the documentation, there is also [HasHelpButton](https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ee684634(v=vs.120)), wdyt about adding it? I have no idea what it would look like

Reply:
And where would the help button lead to?